hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/CallerContext.java

```diff
@@ -114,6 +115,12 @@ public String toString() {
   /** The caller context builder. */
   public static final class Builder {
     private static final String KEY_VALUE_SEPARATOR = ":";
+    /**
+     * The illegal separators include '\t', '\n', '=', etc.
```

Comment:
> , etc.

The illegal separators are only `\t`, `\n`, and `=`.
"etc" is not needed.

Reply:
It's ok. Remove etc from here and other places.